VPC: Error whilst running JDK tests for a built jdk

An error is occuring when the VPC process attempts to trigger tests...

```
rm -f -r /home/vagrant/testLocation/aqa-tests/TKG/../TKG/output_compilation; \
mkdir -p /home/vagrant/testLocation/aqa-tests/TKG/../TKG/output_compilation; \
(ant -f ./scripts/build_tools.xml "-DTEST_JDK_HOME=/home/vagrant/openjdk-build/workspace/build/src/build/linux-x86_64-server-release/images/jdk-25+23
/bin/sh: -c: line 3: unexpected EOF while looking for matching `"'
make: *** [makefile:106: compileTools] Error 2
```

--- a/ansible/pbTestScripts/testJDK.sh
+++ b/ansible/pbTestScripts/testJDK.sh
@@ -7,7 +7,7 @@ if [[ "$(uname)" == "FreeBSD" ]]; then
 	export TEST_JDK_HOME=$HOME/jdk
 else
 	ls -ld $HOME/openjdk-build/workspace/build/src/build/*/images/jdk*
- 	export TEST_JDK_HOME=$(ls -1d $HOME/openjdk-build/workspace/build/src/build/*/images/jdk* |egrep -v 'jre|-image|static-libs|sbom')
+ 	export TEST_JDK_HOME=$(ls -1d $HOME/openjdk-build/workspace/build/src/build/*/images/jdk* |egrep -v 'jre|-image|static-libs|sbom|-jmods')
 fi
 
 echo DEBUG: TEST_JDK_HOME = $TEST_JDK_HOME

--- a/ansible/pbTestScripts/testJDKWin.sh
+++ b/ansible/pbTestScripts/testJDKWin.sh
@@ -12,7 +12,7 @@ rm -rf /cygdrive/c/tmp/*-test-image
 
 # Set Test JDK HOME To The Relocated JDK
 # export TEST_JDK_HOME=C:/cygwin64$(find ~ -maxdepth 1 -type d -name "*jdk*"|grep -v ".*jre"| grep -v ".*-image")
-export TEST_JDK_HOME=`ls -d c:/tmp/jdk*|grep -v "static"|grep -v "debug"|grep -v "jre"|grep -v "test-image"`
+export TEST_JDK_HOME=`ls -d c:/tmp/jdk*|grep -v "static"|grep -v "debug"|grep -v "jre"|grep -v "test-image"|grep -v "jmods"`
 echo TEST_JDK_HOME=$TEST_JDK_HOME
 
 ## Run The Same Tests As Test JDK for Linux

--- a/ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/cygwin/tasks/main.yml
+++ b/ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/cygwin/tasks/main.yml
@@ -13,7 +13,7 @@
     url: https://cygwin.com/setup-x86_64.exe
     dest: C:\temp\cygwin.exe
     force: no
-    checksum: e7815d360ab098fdd1f03f10f43f363c73a632e8866e304c72573cf1e6a0dec8
+    checksum: 2e2e322b138f6337d1a9be71cc0d3b9bb05991c5ac72bd68cf3c1c5f309ecf30
     checksum_algorithm: sha256
   when: not cygwin_installed.stat.exists
   register: cygwin_download

